dpypelines/pipeline/generic_file_ingress_v1.py

```diff
@@ -0,0 +1,184 @@
+import os
+
+from dpytools.http.upload import UploadServiceClient
+from dpytools.logging.logger import DpLogger
+from dpytools.stores.directory.local import LocalDirectoryStore
+
+from dpypelines.pipeline.shared.email_template_message import file_not_found_email
+from dpypelines.pipeline.shared.notification import (
+    BasePipelineNotifier,
+    notifier_from_env_var_webhook,
+)
+from dpypelines.pipeline.shared.pipelineconfig.matching import (
+    get_required_files_patterns,
+)
+from dpypelines.pipeline.shared.utils import (
+    get_email_client,
+    get_submitter_email,
+    str_to_bool,
+)
+
+logger = DpLogger("generic-ingress-pipeline-v1")
```

Comment:
can you change this to just "data-ingress-pipelines" here **and** in dataset_ingress_v1 please.

think of it as the heading that logs will appear under. We probably want all the logs for pipelines under the same heading.

Reply:
Thanks; done!